main.go

```diff
@@ -150,14 +152,19 @@ func main() {
 	syncPeriod := muocfg.SyncPeriodDefault
 
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
-		Namespace:              operatorNS,
-		Scheme:                 scheme,
-		MetricsBindAddress:     metricsAddr,
-		Port:                   9443,
+		Cache: cache.Options{
```

Comment:
I think in this case we have to add `openshift-monitoring` namespace as well. [doc](https://sdk.operatorframework.io/docs/building-operators/golang/operator-scope/#configuring-cluster-scoped-operators-with-multinamespacedcachebuilder) OR if we can keep the `main.go` unchanged works as well
We have faced similar issue earlier: #[OSD-17455](https://issues.redhat.com/browse/OSD-17455)

Reply:
The controller runtime is upgraded to [sigs.k8s.io/controller-runtime](http://sigs.k8s.io/controller-runtime) v0.17.0, with this version, the build failed without changing `ctrl.Options`. I have tested this again with a staging cluster, haven't seen any error message from the log and no `"Service \"prometheus-k8s\" not found"`